app/src/data/schemas/collectionsUUIDSchema.ts

```diff
@@ -0,0 +1,7 @@
+const collectionsUUIDSchema = {
+  response: {
+    // TODO: Awaiting product requirements
```

Comment:
Can you return the comment in the response? That way, when you visit `/api/schema/collections/1234`, it's clear that it's not waiting reqs and not broken. 

`"TODO:": "Awaiting...."` is enough